test/Shared/UserManagerTestBase.cs

```diff
@@ -592,8 +591,8 @@ protected TRole CreateRole(string namePrefix = "")
             {
                 IdentityResultAssert.IsSuccess(await mgr.CreateAsync(u));
             }
-            var usersQ = mgr.Users.Where(u => u.UserName.StartsWith("CanFindUsersViaUserQuerable"));
-            Assert.Null(mgr.Users.FirstOrDefault(u => u.UserName == "bogus"));
+            var usersQ = mgr.Users.Where(u => mgr.GetUserNameAsync(u).Result.StartsWith("CanFindUsersViaUserQuerable"));
+            Assert.Null(mgr.Users.FirstOrDefault(u => mgr.GetUserNameAsync(u).Result == "bogus"));
```

Comment:
Since it seems we are going to keep the `IQueryable` properties it seems better to keep these tests here. I agree we should only have a few predicate building methods like this, but the pattern isn't bad.